packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -181,6 +181,15 @@ export class Blockchain {
   async getBestBlockHash (): Promise<string> {
     return await this.client.call('getbestblockhash', [], 'number')
   }
+
+  /**
+   * Returns details on the active state of the TX memory pool.
+   *
+   * @return {Promise<MempoolInfo>}
+   */
+  async getMempoolInfo (): Promise<MempoolInfo> {
+    return await this.client.call('getmempoolinfo', [], 'number')
```

Comment:
You need to change `number` to `bignumber` in order to get the result in bignumber